automation-roles/40-configure-infra/openshift-cloud-infra/tasks/existing-ocp-cloud-infra.yml

```diff
@@ -0,0 +1,55 @@
+---
+- debug:
+    var: _p_openshift_cluster
+
+- set_fact: 
+    _existing_ocp_infra_type: "standard"
+  when: _p_openshift_cluster.infrastructure.type is not defined
+
+- block:
+  - name: Get Storage Class Information
```

Comment:
@fketelaars ,  run the command "oc get sc -o name" , the oc client will call openshift API to get storage class information. For Extreme case, it could return errors. eg: timeout.  That is the reason, I add the retry.  If the result is good, the command will only run once.